mods/ra/weapons/explosions.yaml

```diff
@@ -12,25 +12,25 @@
 		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
 	Warhead@Smu: LeaveSmudge
 		SmudgeType: Crater
-		InvalidTargets: Structure, Wall, Trees
+		ValidTargets: Ground, Vehicle, Infantry
```

Comment:
To make smudges appear only on empty terrain and below the specified actors. These explosions are for actors blowing up, so `Vehicle` has to be valid as well, otherwise vehicle explosions wouldn't leave/grow craters.